### ISSUE TITLE
translating documents in development

### ISSUE THREAD

**Issue by chaals** [w3c/AB-public#127]
W3C has traditionally not supported translating documents that are in development. In various cases it has even actively opposed this.

The usual rationale given is that there is a real risk of a translation being made of an early version, and not subsequently updated, leaving a particular community who use the translation working from a different document to "the rest of us". This is not a hypothetical concern - we have seen it happen in practice repeatedly.

However, I think the risk is outweighed by the benefits of supporting early translation, particularly in the case where it is done by a person:

- Translating a document in development permits a community we don't normally hear from (those who are not comfortable enough to review a document in english) to participate in the **development** process, potentially providing valuable feedback we would not have got until we believe we are done - when we're much more reluctant to act on it in practice.
- The exercise (especially if done by a person) can reveal that there are concepts or assumptions that are heavily centred in a particular culture and that are very difficult to express in another language. This is important feedback to get before we publish "finalised" content, so the group can consider whether they should reconsider these things.
- Doing a translation means an individual pays close attention to the text, and identify things that are repetitive, complex, etc., which enables them to provide a close review of the original.

Meanwhile, the cost of updating a translation seems to have been significantly reduced by the fact that automatic translators are pretty good. It is wrong to suggest that this means it will automatically happen, but I think it is fair to suggest that the cost of mitigating the risk of outdated information is much lower than it used to be.

**Comment by cwilso** [comment]
I'm not sure what action you're suggesting or looking for, @chaals .  I'm certainly not set against anyone translating documents in development, as long as they are properly identified as works-in-progress.   I don't think, at this point, that the W3C should explicitly fund in-flight translations, though.  (It's not even clear to my how much W3C funds "final" spec translations.)

I do think autotranslation is likely good enough to participate in development in flight.  Are you suggesting we need an explicit milestone of translation?  We have reached out to some of our non-ESL communities and asked for participation in the Vision doc, for example.  Thoughts?

**Comment by chaals** [comment]
I don't think we should require translation, nor am I asking for funding of it (W3C generally doesn't fund any translation of its work).

The proposal is that we explicitly reverse the policy that says "please don't manually translate stuff until it is finished" and welcome people using translation of work in progress as a basis for comment - although we should still be clear that we don't want people to leave a translation of a draft as the last translation available, and set a moral expectation that they commit to translating a recommendation when that is available. (Question arises about groups that just sit in CR...)

**Comment by cwilso** [comment]
Although that's been a de facto reality, does W3C lay that out as policy (or guideline) anywhere?

**Comment by dwsinger** [comment]
perhaps we should start with some best practices on translation? 
* make it clear that the translation is best-effort but the original document is definitive
* make sure that the precise version/date-stamp of the original is documented in the translation
* try to remove or at least mark as outdated translations of the non-current version of the source document
* translations of works in progress should be removed/retired when the document reaches final status (e.g. Recommendation)
* if a document is translated during development, there should be a quality translation of the final document

**Comment by chaals** [comment]
There *are* best practices and policies around translations, and they date back decades. I'll fish out a pointer when I have a little more time.
